turbinia/e2e/run-e2e-gke.sh

```diff
@@ -79,23 +80,36 @@ turbinia-client config list
 echo "Running Turbinia: turbinia-client submit googleclouddisk --project $GCP_PROJECT --zone $GCP_ZONE --disk_name $DISK --request_id $REQUEST_ID"
 turbinia-client submit googleclouddisk --project $GCP_PROJECT --zone $GCP_ZONE --disk_name $DISK --request_id $REQUEST_ID
 
-# Wait until request is received
-req=$(turbinia-client status request $REQUEST_ID -j)
-while [[ -z "$req" ]]
+# Wait until request is running
+req_status=$(turbinia-client status request $REQUEST_ID -j | jq -r '.status')
+RETRIES=0
+while [[ $req_status != "running" ]]
 do
-  echo "Request $REQUEST_ID is still populating. Sleeping for 5 seconds..."
+  RETRIES-=1
```

Comment:
Should this be `RETRIES+=1` seems like this would go negative?

Reply:
Doh. Yes, thanks Wajih :)